akeyless_gateway/assets/dashboards/akeyless_gateway_dashboard.json

```diff
@@ -1,873 +1,879 @@
 {
-   "title":"Akeyless GW",
-   "description":"The [Akeyless Vaultless Platform](https://www.akeyless.io) is a unified secrets management system that enables you to store, protect, rotate, and dynamically create credentials, certificates, and encryption keys.\n\n### [Official Akeyless docs](https://docs.akeyless.io/)\n### [Monitoring Akeyless With Datadog](https://docs.akeyless.io/) (cloned)",
-   "widgets":[
-      {
-         "id":15,
-         "definition":{
-            "type":"image",
-            "url":"https://camo.githubusercontent.com/bc81eb17df70339e2c7ce70ae7d13a13abebf182cc9a3f672b1d43161f79e1ec/68747470733a2f2f7777772e616b65796c6573732e696f2f77702d636f6e74656e742f75706c6f6164732f323032312f30332f616b65796c6573732d6c6f676f2d626c61636b2d7472616e73706172656e742e706e67",
-            "sizing":"scale-down",
-            "margin":"lg",
-            "has_background":true,
-            "has_border":true,
-            "vertical_align":"center",
-            "horizontal_align":"center"
-         }
-      },
-      {
-         "id":7543139320471660,
-         "definition":{
-            "type":"note",
-            "content":"The [Akeyless Vaultless Platform](https://www.akeyless.io) is a unified secrets management system that enables you to store, protect, rotate, and dynamically create credentials, certificates, and encryption keys.\n\n### [Official Akeyless docs](https://docs.akeyless.io/)\n### [Monitoring Akeyless With Datadog](https://docs.akeyless.io/docs/telemetry-metrics)",
-            "background_color":"vivid_blue",
-            "font_size":"14",
-            "text_align":"left",
-            "vertical_align":"center",
-            "show_tick":true,
-            "tick_pos":"50%",
-            "tick_edge":"left",
-            "has_padding":true
-         }
-      },
-      {
-         "id":4,
-         "definition":{
-            "title":"Running containers",
-            "title_size":"16",
-            "title_align":"center",
-            "time":{
-               "live_span":"1m"
+  "title": "Akeyless GW",
+  "description": "The [Akeyless Vaultless Platform](https://www.akeyless.io) is a unified secrets management system that enables you to store, protect, rotate, and dynamically create credentials, certificates, and encryption keys.\n\n### [Official Akeyless docs](https://docs.akeyless.io/)\n### [Monitoring Akeyless With Datadog](https://docs.akeyless.io/)",
+  "widgets": [
+    {
+      "id": 15,
+      "definition": {
+        "type": "image",
+        "url": "https://www.akeyless.io/wp-content/uploads/2024/04/akeyless-gw-logo-light.png",
```

Comment:
If this is erroring, can you try updating it with: https://raw.githubusercontent.com/DataDog/integrations-extras/cce817278ad2916be89f5c012294e4b5446760c0/akeyless_gateway/assets/logos/akeyless-gw-logo-light.png 

Lmk if that resolves the issue

Reply:
I don't see anything actually updated or changed in prod...

Reply:
This won't change in prod until we publish the integration. You can use your sandbox account to update the widget image or update the json here. In your sandbox, you can then see if the image shows correctly

Reply:
I am logged into what I believe is my sandbox account, but it still shows as empty. Unless I'm not understanding how to check the change (using the DPN account).

Reply:
Hey @akljph did you update the widget in your sandbox with the link above? it looks like the widget is still using the old image link. Can you try updating the widget with the new link and let me know if the image appears after saving?

Reply:
Yes. If I clone the dashboard and add the link, it does work.

Reply:
Great! Can you replace the current image with the one that works in the cloned dashboard in the sandbox account?

Reply:
This one: https://raw.githubusercontent.com/DataDog/integrations-extras/cce817278ad2916be89f5c012294e4b5446760c0/akeyless_gateway/assets/logos/akeyless-gw-logo-light.png

Reply:
Done!